src/EPR.PRN.Backend.Data/Migrations/20240728110950_AddedEnumForStatus.cs

```diff
@@ -14,7 +14,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
         {
             migrationBuilder.InsertData(
                 table: "PrnStatus",
-                columns: new[] { "Id", "StatusDescription", "StatusName" },
+                columns: ["Id", "StatusDescription", "StatusName"],
```

Comment:
Not sure this will work. It is trying to initialise the array. doing it this way wont work

Reply:
it will work.
it's a suggestion by git / vs